publication/ver11/societal_impact.md

```diff
@@ -0,0 +1,239 @@
+However, since WoT uses JSON-LD, it would be possible in theory to annotate WoT
+Thing Descriptions with information such as this.
+
+In general, additional metadata can be attached to WoT Thing Descriptions and
+this can be used for arbitrary purposes that can go beyond what is currently
+in the specification.  For example, a given device manufacturer could attach
+geolocation information to a WoT Thing Description.  To mitigate privacy
+hazards from both current and future metadata in WoT Thing Descriptions, the 
+WoT Discovery process provides mechanisms to control access to WoT Thing Descriptions.
+
+### 3. [What risks do you see in features of your specification being misused, or used differently from how you intended?](https://w3ctag.github.io/societal-impact-questionnaire/#misuse)
+Many possible misuses and risks are already discussed in the 
+[answers to the Security and Privacy questionnaire](https://github.com/w3c/wot-architecture/blob/main/publication/ver11/security_and_privacy.md), 
+such as distributed denial of service attacks and
+man-in-the-middle attacks.  See also the individual specifications for 
+descriptions of risks and mitigations.  
+
+In general, in addition to the "normal" risks of access to digital information,
+IoT devices can also provide real-time sensor data (a surveillance risk), and
+IoT actuators could potentially be a safety risk (e.g. an attacker starting a car in
+an enclosed space, filling the garage with carbon monoxide).  Most of these
+already exist without WoT, so what WoT brings is easier access to this functionality.
+It describes what already exists, it does not create it.  Mechanisms provided
+in WoT Discovery, however, can be used to limit access to WoT metadata to those
+who already have access to the associated IoT services.
+
+### 4. [Can users of the Web Platform choose not to use features of your specification?](https://w3ctag.github.io/societal-impact-questionnaire/#opt-out)
+An end user can choose not to install IoT devices.
+They could also use to access IoT services directly without first accessing
+WoT metadata (e.g. using specific documentation for that IoT service)
+although this would in most cases lead to significantly more effort.
+If an application needs WoT metadata to function, availability and
+use of WoT metadata would, of course, be required.  Applications can however
+be designed to avoid the use of WoT metadata if necessary, at least for
+specific IoT devices.
+
+A device manufacturer can also choose not to publish WoT metadata for their
+devices.  However, this does not prevent others from publishing WoT metadata
+about their devices.  This is however similar to the fact that web content
+in general can be published about any public subject.
+ 
+### 5. [What groups of people are excluded from using features of your specification?](https://w3ctag.github.io/societal-impact-questionnaire/#excluded)
+No specific groups of people are excluded, although language may be a barrier.
```

Comment:
I would not consider the language barrier here.

Rather the limited set of mechanisms to describe human interaction / accessible UIs might impact some people, see also the discussion in Profiles about support for people with disabilities.

Reply:
I'm not sure if I understand this comment. TD has mechanisms onboard that can be used for UI use cases (e.g., "title", "description", etc.). The semantic annotations features help to provide context knowledge.